diff --git a/vte/share/gir-1.0/Vte-2.91.gir b/vte/share/gir-1.0/Vte-2.91.gir
index ba1f3d6..e657a5c 100644
--- a/vte/share/gir-1.0/Vte-2.91.gir
+++ b/vte/share/gir-1.0/Vte-2.91.gir
@@ -1706,7 +1706,7 @@ property by ID. See that function for more information.</doc>
           </parameter>
         </parameters>
       </method>
-      <method name="ref_property_uri"
+      <!--<method name="ref_property_uri"
               c:identifier="vte_properties_ref_property_uri"
               version="0.86">
         <doc xml:space="preserve"
@@ -1734,8 +1734,8 @@ property by ID. See that function for more information.</doc>
             <type name="utf8" c:type="const char*"/>
           </parameter>
         </parameters>
-      </method>
-      <method name="ref_property_uri_by_id"
+      </method>-->
+      <!--<method name="ref_property_uri_by_id"
               c:identifier="vte_properties_ref_property_uri_by_id"
               version="0.86">
         <doc xml:space="preserve"
@@ -1763,7 +1763,7 @@ by ID. See that function for more information.</doc>
             <type name="gint" c:type="int"/>
           </parameter>
         </parameters>
-      </method>
+      </method>-->
       <method name="ref_property_variant"
               c:identifier="vte_properties_ref_property_variant"
               version="0.86">
