diff --git a/zoha-vte/src/auto/terminal.rs b/zoha-vte/src/auto/terminal.rs
index 2eade10..906c518 100644
--- a/zoha-vte/src/auto/terminal.rs
+++ b/zoha-vte/src/auto/terminal.rs
@@ -14,10 +14,11 @@ use glib::{
     translate::*,
 };
 use std::boxed::Box as Box_;
+use std::ffi::c_char;
 
 glib::wrapper! {
     #[doc(alias = "VteTerminal")]
-    pub struct Terminal(Object<ffi::VteTerminal, ffi::VteTerminalClass>) @extends gtk::Widget, gobject::InitiallyUnowned, @implements atk::ImplementorIface, gtk::Buildable, gtk::Scrollable;
+    pub struct Terminal(Object<ffi::VteTerminal, ffi::VteTerminalClass>) @extends gtk::Widget, glib::InitiallyUnowned, @implements atk::GObjectAccessible, gtk::Buildable, gtk::Scrollable;
 
     match fn {
         type_ => || ffi::vte_terminal_get_type(),
@@ -661,19 +662,36 @@ pub trait TerminalExt: IsA<Terminal> + 'static {
         regexes: &[&Regex],
         match_flags: u32,
     ) -> Option<Vec<glib::GString>> {
-        unsafe {
-            let mut n_regexes = std::mem::MaybeUninit::uninit();
-            let mut matches = Vec::<glib::GString>::uninitialized();
-            let ret = from_glib(ffi::vte_terminal_event_check_regex_simple(
+        let n_regexes = regexes.len();
+        let mut matches: Vec<*mut c_char> = vec![std::ptr::null_mut(); regexes.len()];
+
+        let ok: bool = unsafe {
+            from_glib(ffi::vte_terminal_event_check_regex_simple(
                 self.as_ref().to_glib_none().0,
                 event.to_glib_none_mut().0,
                 regexes.to_glib_none().0,
-                n_regexes.as_mut_ptr(),
+                n_regexes,
                 match_flags,
-                matches.to_glib_none_mut().0,
-            ));
-            if ret { Some(matches) } else { None }
+                matches.as_mut_ptr(),
+            ))
+        };
+
+        if !ok {
+            return None;
         }
+
+        let conv: Vec<glib::GString> = matches
+            .into_iter()
+            .filter_map(|ptr| {
+                if ptr.is_null() {
+                    None
+                } else {
+                    Some(unsafe { from_glib_full(ptr) })
+                }
+            })
+            .collect();
+
+        return Some(conv);
     }
 
     #[doc(alias = "vte_terminal_feed")]
@@ -2418,7 +2436,7 @@ pub trait TerminalExt: IsA<Terminal> + 'static {
     #[doc(alias = "vte_terminal_watch_child")]
     fn watch_child(&self, child_pid: glib::Pid) {
         unsafe {
-            ffi::vte_terminal_watch_child(self.as_ref().to_glib_none().0, child_pid);
+            ffi::vte_terminal_watch_child(self.as_ref().to_glib_none().0, child_pid.into_glib());
         }
     }
 
@@ -2942,36 +2960,7 @@ pub trait TerminalExt: IsA<Terminal> + 'static {
         &self,
         f: F,
     ) -> SignalHandlerId {
-        unsafe extern "C" fn termprops_changed_trampoline<
-            P: IsA<Terminal>,
-            F: Fn(&P, &[i32], i32) -> bool + 'static,
-        >(
-            this: *mut ffi::VteTerminal,
-            props: /*Unimplemented*/ gpointer,
-            n_props: std::ffi::c_int,
-            f: glib::ffi::gpointer,
-        ) -> glib::ffi::gboolean {
-            unsafe {
-                let f: &F = &*(f as *const F);
-                f(
-                    Terminal::from_glib_borrow(this).unsafe_cast_ref(),
-                    &from_glib_none(props),
-                    n_props,
-                )
-                .into_glib()
-            }
-        }
-        unsafe {
-            let f: Box_<F> = Box_::new(f);
-            connect_raw(
-                self.as_ptr() as *mut _,
-                c"termprops-changed".as_ptr() as *const _,
-                Some(std::mem::transmute::<*const (), unsafe extern "C" fn()>(
-                    termprops_changed_trampoline::<Self, F> as *const (),
-                )),
-                Box_::into_raw(f),
-            )
-        }
+        unimplemented!()
     }
 
     #[doc(alias = "allow-hyperlink")]
diff --git a/zoha-vte/src/auto/uuid.rs b/zoha-vte/src/auto/uuid.rs
index 5cb1e62..9dbd296 100644
--- a/zoha-vte/src/auto/uuid.rs
+++ b/zoha-vte/src/auto/uuid.rs
@@ -58,7 +58,7 @@ impl Uuid {
             from_glib_full(ffi::vte_uuid_free_to_string(
                 self.into_glib_ptr(),
                 fmt.into_glib(),
-                len,
+                std::ptr::null_mut(),
             ))
         }
     }
